example/cds-hooks-api-guide/src/prefetch-context/prefetch-context.js

```diff
@@ -7,117 +7,118 @@ import {
   newAppointment,
 } from "./util.js";
 
-export default new Service(
-  {
-    id: "prefetch-context",
-    title: "Patient view with last encounter.",
-    hook: "patient-view",
-    description:
-      "A patient-view hook with patient and encounter prefetch template values. Presents patient info and last encounter information",
-    prefetch: {
-      patient: "Patient/{{context.patientId}}",
-      encounter: "Encounter?subject={{context.patientId}}&_sort=date",
-    },
+const options = {
+  id: "prefetch-context",
+  title: "Patient view with last encounter.",
+  hook: "patient-view",
+  description:
+    "A patient-view hook with patient and encounter prefetch template values. Presents patient info and last encounter information",
+  prefetch: {
+    patient: "Patient/{{context.patientId}}",
+    encounter: "Encounter?subject={{context.patientId}}&_sort=date",
```

Comment:
This would be a _lot_ better, especially because there are a lot of different _sort values (and also because depending on the resource there are quite a few different keys, and they differ depending on the resource).